ModelModel/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  ModelModel
+//
+//  Created by Stanislav Cherkasov on 12.12.17.
+//  Copyright © 2017 Stanislav Cherkasov. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
```

Comment:
You can remove all this methods, they all have empty implementations and add to your project complexity